openlibrary/templates/support.html

```diff
@@ -16,12 +16,12 @@ <h1>$_("How can we help?")</h1>
 
  <div class="formElement">
  <div class="label"><label for="name">$_("Your Name")</label></div>
- <div class="input"><input type="text" class="input" name="name" id="name"/></div>
+ <div class="input,required" ><input type="text" class="input" name="name" id="name"/></div>
```

Comment:
Please revert this.  It is unrelated to the email validation.

As an aside, the list of classes in the `class` attribute must be separated by spaces, not commas.

Reply:
@jamesachamp reverted. Thanks for the tip about `class` attribute